### PR TITLE
Fixed unsatisfied dependency

### LINKED ISSUE #207: RGB-LED won't work with the published library
**Describe the bug** \

> There is a dependency issue for RGB led due to MultiPinConfiguration

**What component are you facing this issue with?** \
The component details and what functionality causes this issue

> Initializing RGB-LED

**To Reproduce** \
Steps to reproduce the behavior: For example

> Implement the RGB led using the published library.

**Expected behavior** \

> RGB should initialize and work as expected

**Screenshots** \

image

### PR DESCRIPTION
**Pull request by ruthvikm**
Fixed unsatisfied dependency for multipin components. Closes #207

### CODE REVIEW

**Review by adelgadoj3 — APPROVED** [review]
Looks good to me. Everything looks clear and concise.

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/utilities/DigitalInputConfiguration.java [hunk 1]
```diff
@@ -53,12 +53,12 @@ public void setName(String name) {
 
     /**
      * Gets the current debounce value for the component.
-     * @return Long type representing the debounce of the component.
+     * @return Long type representing to debounce of the component.
```
Comment (joefol) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/utilities/DigitalInputConfiguration.java:56]: Is this grammatically correct?

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/utilities/DigitalInputConfiguration.java [hunk 2]
```diff
@@ -53,12 +53,12 @@ public void setName(String name) {
...
      */
     public Long getDebounce() { return debounce; }
 
     /**
-     * Sets the debounce of the component. Replaces existing debounce.
+     * Sets to debounce of the component. Replaces existing debounce.
```
Comment (joefol) [inline pi4micronaut-utils/src/main/java/com/opensourcewithslu/utilities/DigitalInputConfiguration.java:61]: Same here

**Review by joefol — CHANGES_REQUESTED** [review]
Everything else looks good!

File: pi4micronaut-utils/src/main/java/com/opensourcewithslu/utilities/DigitalInputConfiguration.java (hunk 1 above)
Comment (ruthvikm) [reply pi4micronaut-utils/src/main/java/com/opensourcewithslu/utilities/DigitalInputConfiguration.java:56]: This was suggested by Javadoc parser

**Review by SwindleA — APPROVED** [review]
👍

**Review by joefol — APPROVED** [review]
Looks good!

**Pull request merged**